docs/devconsole/setup-gitops.md

```diff
@@ -0,0 +1,80 @@
+## Steps to follow for seting up a sample GitOps pipelines using [kam](../../releases)
+
+- Complete all the pre-requisite steps mentioned in the [docs](/docs/journey/day1#day-1-operations)
+- Bootstrap a GitOps repository following steps mentioned in the [docs](/docs/journey/day1#bootstrapping-the-manifest)
+
+  ```sh
+  $ kam bootstrap --service-repo-url https://github.com/<username>/taxi.git \          
+    --dockercfgjson ~/Downloads/<username>-gitops-auth.json \
+    --gitops-repo-url https://github.com/<username>/gitops-example.git \
+    --image-repo quay.io/<username>/taxi --prefix demo --output <output directory> 
```

Comment:
Can you drop the `--prefix` here? it's optional and adds complications to all the other output.

Reply:
Removed.   @bigkevmcd pls try the lgtm command to see if it works.